pandas/_libs/tslibs/util.pxd

```diff
@@ -2,23 +2,13 @@
 from cpython cimport PyTypeObject
 
 cdef extern from *:
-    """
-    PyObject* char_to_string(const char* data) {
-    #if PY_VERSION_HEX >= 0x03000000
-        return PyUnicode_FromString(data);
-    #else
-        return PyString_FromString(data);
-    #endif
-    }
-    """
     object char_to_string(const char* data)
```

Comment:
shouldnt L14 be removed too?

Reply:
It's actually used here:

https://github.com/pandas-dev/pandas/blob/2d65e38f5c245a8410c7cb37ec17424def00fa78/pandas/_libs/tslibs/period.pyx#L1267

So maybe but would require a more comprehensive refactor

Reply:
I think that just becomes `result = PyUnicode_FromString(formatted)`

Reply:
Cool - I'll give it a look. Seems like this caused a somewhat surprising failure so that might solve

Reply:
Hmm so didn't realize the cdef of the tripled quoted string was actually using that as a macro. Was thinking of cimporting PyUnicode_FromString directly but looks like that isn't available until Cython 3.0.0

https://github.com/cython/cython/pull/3010

For now going to revert to macro